css/login.css

```diff
@@ -0,0 +1,164 @@
+body{
+	background-color: #555555;
+	margin: 0;
+	font-family: 'Roboto', sans-serif;
+	min-width: 1200px;
+}
+
+.rectangle{
+	width: 100%;
+	height: 640px;
+}
+
+.firstRectangle{
+	background-image: url('../img/feed.jpg');
+	background-blend-mode: overlay;
+}
+
+.whiteRectangle{
+	width: inherit;
+	height: inherit;
+	background-color: #CBF6FF;
+	opacity: 0.3;
+	position: absolute;
+}
+
+.logInBox{
+	width: 375px;
+	height: 400px;
+	margin-left: 400px;
+	background-color: #F5F5F5;
+	opacity: 1!important;
+	position: absolute;
+	margin-top: 140px;
+	box-shadow: 0px 0px 8px black;
+	border-radius: 3px;
+}
+
+.headLogIn{
+	width: inherit;
+	background-color: #1ab2ff;
+	height: 75px;
+	box-shadow: 0px 3px 10px grey;
+	display: table;
+}
+
+.headLogIn > div{
+	color: white;
+	height: 75px;
+	font-size: 26px;
+	text-align: center;
+	line-height: 75px;
+}
+
+.contentLogIn{
+	width: 90%;
+	height: 70%;
+	margin: 0 auto;
+	margin-top: 20px;
+}
+
+.logFormClass{
+	margin-top: 20px;
+	position: absolute;
+	width: inherit;
+	background-color: #F5F5F5;
+	text-align: center;
+}
+
+.logFormInput{
+	border: 0;
+	width: 100%;
+	height: 50px;
+	margin-bottom: 10px;
+	background-color: inherit;
+	padding-left: 10px;
+	font-size: 15px;
+	border-bottom: 1px solid grey;
+}
+
+.logFormButton{
+	margin: 0 auto;
+	width: 120px;
+	height: 30px;
+	background-color: #1ab2ff;
+	color: white;
+	margin-top: 50px;
+	border: 0;
+	box-shadow: 0px 3px 15px grey;
+}
+
+/* Second Rectangle */
+
+.secondRectangle{
+	background-color: white;
+	height: 450px;
+}
+
+.imgContainer{
+	width: 375px;
+	height: 375px;
+	margin-left: 400px;
+	float: left;
+	margin-right: 150px;
+}
+
+.imgContainer img{
+	width: 100%;
+	height: 100%;
+}
+
+.textContainer{
+	padding: 50px 20px  10px  10px;
+}
+
+.tcHead{
+	font-size: 28px;
+	font-weight: 500;
+	margin-bottom: 80px;
+}
+
+.textContainer{
+	margin-left: 200px;
+}
+
+.tcBody{
+	margin-right: 300px;
+}
+
+.thirdRectangle{
+	background-color: lightgrey;
+	height: 470px;
+}
+
+.textContainerThird{
+	margin-left: 400px;
+	margin-top: 50px;
+	width: 500px;
+	position: absolute;
+}
+
+.imgContainerthird{
+	float: right;
+	margin-right: 300px;
+}
+
+.imgContainerthird img{
+	height: 250px;
+	margin-top: 100px;
+}
+
+.footer{
+	clear: both;
+	width: 100%;
+	background-color: #bababa;
+	height: 100px;
+	color: #4d4d4d;
+}
+
+.footer h3{
```

Comment:
Throughout the other sections of the page, that would indeed make sense semantically. But for the footer, I'm not sure I understand. It would be like adding a subtitle at the bottom of the page. Are you sure that @zuldy was referring to the page footer?